tests/extensions/inline/italic/full/custom-attribute.rng

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Having a separate folder like `tests/extensions/inline/italic/full` for each extension promotes decoupling and isolation of these tests. It may not scale very well however if a dozen files are needed for each element, so we could move the trade-off towards test grouping by creating a scenario (`tests/thedailybugle/typical-inline-elements/`)?

Reply:
I do wonder that. Let's see how it goes though.